borg/testsuite/benchmark.py

```diff
@@ -0,0 +1,105 @@
+"""
+Do benchmarks using pytest-benchmark.
+
+Usage:
+
+    py.test --benchmark-only
+"""
+
+import os
+
+import pytest
+
+from .archiver import changedir, exec_cmd
+
+
+@pytest.fixture
+def cmd():
```

Comment:
I want that to be able to use other borg code later - testing python based, testing binary via fixture params.
in fact, I already had that code, just removed it to do it in a separate commit / pr.